debian/postinst

```diff
@@ -37,7 +37,9 @@ rsync -a "/$modpath/vdso" "./$modpath/"
 rsync -a "/$modpath/extra" "./$modpath/"
 rsync -a "/$modpath/kernel/drivers" "./$modpath/kernel/"
 
-if [[ $(cat /var/lib/delphix-appliance/platform) == "gcp" ]]; then
+if [[ $(cat /var/lib/delphix-appliance/platform) == "gcp" || \
```

Comment:
Why we only copy the network drivers for those particular platforms?

Reply:
Specifically we only copy the kernel core drivers for these platforms because they aren't required on other platforms. One goal for the recovery environment is to keep the package small and lightweight as much as possible. If we don't care as much about that goal anymore, we could copy in more kernel modules to attempt to reduce the risk of similar issues occurring in the future.

Reply:
If we were to add support for a new platform in the future that needed network drivers, do we have tests that would catch this issue?

I think we may want to do something like:
```
case "$(cat /var/lib/delphix-appliance/platform)" in
gcp | kvm | oci) rsync -a "/$modpath/kernel/net/" "./$modpath/kernel/net/" ;;
esx | aws | azure | hyperv) echo "skipping copying of drivers in $modpath/kernel/net/" ;;
*) <throw some error>
esac
```
This will force us to re-evaluate whether or not the new platform needs those drivers.